code/modules/jobs/job_types/roguetown/courtier/jester.dm

```diff
@@ -46,6 +45,14 @@
 		H.STAINT = rand(1, 20)
 		H.STALUC = rand(1, 20)
 		H.cmode_music = 'sound/music/combat_jester.ogg'
+		// Mime vs Jester. 
+		// As a mute jester you cannot cast Tell Joke/Tragedy, so why even have them?
+		if(HAS_TRAIT(H, TRAIT_PERMAMUTE)) // I considered adding a check for Xylix patron but in the off chance there's a mute non-xylix jester I don't want to fuck them over.
+			H.mind.AddSpell(new /obj/effect/proc_holder/spell/aoe_turf/conjure/mime_wall)
+			H.mind.AddSpell(new /obj/effect/proc_holder/spell/aoe_turf/conjure/mime_chair)
```

Comment:
please go through the spells themselves to make sure there's nothing goofy and update any spans you might find